ste/mgr-JobPartMgr.go

```diff
@@ -90,7 +90,7 @@ const AzCopyMaxIdleConnsPerHost = 500
 func NewAzcopyHTTPClient() *http.Client {
 	return &http.Client{
 		Transport: &http.Transport{
-			Proxy: http.ProxyFromEnvironment,
+			Proxy: getProxy(),
```

Comment:
This is not the only place in our codebase where HTTP clients are created.  Need to find the others, and fix them too.

Reply:
The only other places are for the internal RPC (which I don't think _needs_ a proxy necessarily?), unless said clients are created inside the blob and file SDKs.